vendor/github.com/ethereum/go-ethereum/les/backend.go

```diff
@@ -249,6 +249,7 @@ func (s *LightEthereum) Stop() error {
 	s.eventMux.Stop()
 
 	time.Sleep(time.Millisecond * 200)
+	s.wg.Wait()
```

Comment:
this is called in the protocolManager.Stop, not very obvious but this wg is passed to protocol manager on creation)

Reply:
it's not that anyway, reproduced it locally